sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -350,6 +350,7 @@ export class ApplicationController extends BaseController {
       fullName: getUserFullName(application.student.user),
       programName: application.offering.educationProgram.name,
       locationName: application.location.name,
+      offeringIntensity: application.offering.offeringIntensity,
```

Comment:
What about the form.io definition ?

Reply:
This is part of the PR @dheepak-aot , will test the api and change the formio and will create one more PR

Reply:
Thanks for explaining. I would recommend include that in description of the PR because I wasn't sure if form definition was in the scope or not.